shell/src/main/java/org/apache/accumulo/shell/commands/ClasspathCommand.java

```diff
@@ -45,4 +45,22 @@ public String description() {
   public int numArgs() {
     return 0;
   }
+
+  public static void printClassPath(PrintWriter writer) {
+    try {
+      writer.print("Accumulo Shell Classpath: \n");
+
+      final String javaClassPath = System.getProperty("java.class.path");
+      if (javaClassPath == null) {
+        throw new IllegalStateException("java.class.path is not set");
+      }
+      Arrays.stream(javaClassPath.split(File.pathSeparator)).forEach(classPathUri -> {
+        writer.print(classPathUri + "\n");
+      });
+
+      writer.print("\n");
+    } catch (Exception t) {
```

Comment:
I would just catch the checked exceptions here using multi-catch syntax, so you don't rewrap RTEs.